### PR TITLE
Redirecting /account/books/[^/]* to /people/user/books/

### LINKED ISSUE #3461: Redirect /account/books to /people/our-username
A good chunk of people try sharing their /account/books page. This will enable us to move in the direction of `following` readers when we try to do that, and also redirectly the users would get us into a sweet spot which enable us to move in a variety of directions.

### Describe the problem that you'd like solved

We should consider changing the URL /people/our-username to render your editable /account/books page if we're logged in and that's us otherwise, show the public-readable version of the page.

### Proposal & Constraints

https://www.goodreads.com/user/show/115149571-tabish-shaikh

### Stakeholders

@mekarpeles

### PR DESCRIPTION
**Pull request by tabshaikh**
Closes #3461 

### Technical

**Redirection:**
/account/books/[^/]* -> /people/user/books/
/account/books -> /people/user
**page/user edited to add manage privacy button**
### Testing

Visit the /people/user/books page:
- [ ] Logged in
- [ ] Not logged in

Visit the /people/user page:
- [ ] Logged in
- [ ] Not logged in

### Stakeholders

@mekarpeles

### CODE REVIEW

File: openlibrary/templates/type/user/view.html [hunk 1]
```diff
@@ -40,10 +40,19 @@ <h1>$page.displayname</h1>
...
+    $if user:     
+        $if is_public:
+            <p>$:_('You have chosen to publicly share the books they are <a href="%(username)s/books/currently-reading">currently reading</a>, <a href="%(username)s/books/already-read">have already read</a>, and <a href="%(username)s/books/want-to-read">want to read</a>!', username=page.key)</p>
+        $else:
+            <span>$_('You have chosen to keep your Reading Log private')</span>
+        <a href="/account/privacy">$_('Manage your privacy settings')</a>
     $else:
-      <p>$_('This user has chosen to keep their Reading Log private')</p>
+        $if is_public:
+            <p>$:_('This user has chosen to publicly share the books they are <a href="%(username)s/books/currently-reading">currently reading</a>, <a href="%(username)s/books/already-read">have already read</a>, and <a href="%(username)s/books/want-to-read">want to read</a>!', username=page.key)</p>
```
Comment (mekarpeles) [inline openlibrary/templates/type/user/view.html:53]:
```suggestion
            <p>$:_('Here are the books %(username)s is <a href="%(username)s/books/currently-reading">currently reading</a>, <a href="%(username)s/books/already-read">have already read</a>, and <a href="%(username)s/books/want-to-read">want to read</a>!', username=page.key)</p>
```

Comment (mekarpeles) [reply openlibrary/templates/type/user/view.html:53]: I need to test this, but I think this copy is the right direction

Comment (tabshaikh) [reply openlibrary/templates/type/user/view.html:53]: That shows /people/openlibrary and page.displayname is correct as shown below:
```<p>$:_('Here are the books %(user)s is <a href="%(username)s/books/currently-reading">currently reading</a>, <a href="%(username)s/books/already-read">have already read</a>, and <a href="%(username)s/books/want-to-read">want to read</a>!', username=page.key, user=page.displayname)</p>```

Comment (tabshaikh) [reply openlibrary/templates/type/user/view.html:53]: Pushing this change

File: openlibrary/templates/type/user/view.html [hunk 2]
```diff
@@ -40,10 +42,17 @@ <h1>$page.displayname</h1>
...
+            <p>$:_('You have chosen to publicly share the books they are <a href="%(username)s/books/currently-reading">currently reading</a>, <a href="%(username)s/books/already-read">have already read</a>, and <a href="%(username)s/books/want-to-read">want to read</a>!', username=page.key)</p>
+        $else:
+            <span>$_('You have chosen to keep your Reading Log private')</span>
+        <a href="/account/privacy">$_('Manage your privacy settings')</a>
     $else:
-      <p>$_('This user has chosen to keep their Reading Log private')</p>
+        $if is_public:
+            <p>$:_('Here are the books %(user)s is <a href="%(username)s/books/currently-reading">currently reading</a>, <a href="%(username)s/books/already-read">have already read</a>, and <a href="%(username)s/books/want-to-read">want to read</a>!', username=page.key, user=page.displayname)</p>
+        $else:
+            <p>$_('This user has chosen to keep their Reading Log private')</p>
```
Comment (tabshaikh) [inline openlibrary/templates/type/user/view.html:55]:
```suggestion
            <p>$_('This user has chosen to keep their')<a href="/account/books">$_('Reading Log')</a> $_('private')</p>
```

File: openlibrary/templates/type/user/view.html [hunk 3]
```diff
@@ -40,10 +42,17 @@ <h1>$page.displayname</h1>
 
 
     <h2>$_('Reading Log')</h2>
-    $if settings and settings.get('public_readlog', 'no') == "yes":
-      <p>$:_('This user has chosen to publicly share the books they are <a href="%(username)s/books/currently-reading">currently reading</a>, <a href="%(username)s/books/already-read">have already read</a>, and <a href="%(username)s/books/want-to-read">want to read</a>!', username=page.key)</p>
+    $if owners_page:     
+        $if is_public:
+            <p>$:_('You have chosen to publicly share the books they are <a href="%(username)s/books/currently-reading">currently reading</a>, <a href="%(username)s/books/already-read">have already read</a>, and <a href="%(username)s/books/want-to-read">want to read</a>!', username=page.key)</p>
+        $else:
+            <span>$_('You have chosen to keep your Reading Log private')</span>
```
Comment (tabshaikh) [inline openlibrary/templates/type/user/view.html:49]:
```suggestion
            <span>$_('You have chosen to keep your') <a href="/account/books">$_('Reading Log')</a> $_('private')</span>

```

File: openlibrary/templates/type/user/view.html [hunk 4]
```diff
@@ -40,10 +42,17 @@ <h1>$page.displayname</h1>
...
+        <a href="/account/privacy">$_('Manage your privacy settings')</a>
     $else:
-      <p>$_('This user has chosen to keep their Reading Log private')</p>
+        $if is_public:
+            <p>$:_('Here are the books %(user)s is <a href="%(username)s/books/currently-reading">currently reading</a>, <a href="%(username)s/books/already-read">have already read</a>, and <a href="%(username)s/books/want-to-read">want to read</a>!', username=page.key, user=page.displayname)</p>
+        $else:
+            <p>$_('This user has chosen to keep their') <a href="/account/books">$_('Reading Log')</a> $_('private')</p>
```
Comment (mekarpeles) [inline openlibrary/templates/type/user/view.html:55]:
```suggestion
            <p>$_('This reader has chosen to make their Reading Log private.')</p>
```

File: openlibrary/templates/type/user/view.html [hunk 5]
```diff
@@ -40,10 +42,18 @@ <h1>$page.displayname</h1>
 
 
     <h2>$_('Reading Log')</h2>
-    $if settings and settings.get('public_readlog', 'no') == "yes":
-      <p>$:_('This user has chosen to publicly share the books they are <a href="%(username)s/books/currently-reading">currently reading</a>, <a href="%(username)s/books/already-read">have already read</a>, and <a href="%(username)s/books/want-to-read">want to read</a>!', username=page.key)</p>
+    $if owners_page:     
+        $if is_public:
+            <p>$:_('You have chosen to publicly share the books they are <a href="%(username)s/books/currently-reading">currently reading</a>, <a href="%(username)s/books/already-read">have already read</a>, and <a href="%(username)s/books/want-to-read">want to read</a>!', username=page.key)</p>
```
Comment (mekarpeles) [inline openlibrary/templates/type/user/view.html:47]:
```suggestion
            <p>$:_('You are publicly sharing the books you are <a href="%(username)s/books/currently-reading">currently reading</a>, <a href="%(username)s/books/already-read">have already read</a>, and <a href="%(username)s/books/want-to-read">want to read</a>.', username=page.key)</p>
```

**Pull request merged**